rollup.config.js

```diff
@@ -39,50 +44,37 @@ const runtime_entrypoints = Object.fromEntries(
  * @type {import("rollup").RollupOptions[]}
  */
 export default [
+	// Runtime: Generates CJS builds and type definition entry points for svelte/* and generated internal_exports.js
 	{
 		input: {
 			...runtime_entrypoints,
 			index: 'src/runtime/index.js',
 			ssr: 'src/runtime/ssr.js'
 		},
-		output: ['es', 'cjs'].map(
-			/** @returns {import('rollup').OutputOptions} */
-			(format) => {
-				const ext = format === 'es' ? 'mjs' : 'js';
-				return {
-					entryFileNames: (entry) => {
-						if (entry.isEntry) {
-							if (entry.name === 'index') return `index.${ext}`;
-							else if (entry.name === 'ssr') return `ssr.${ext}`;
+		output: {
+			entryFileNames: (entry) => {
+				if (entry.isEntry) {
+					if (entry.name === 'index') return `index.cjs`;
+					else if (entry.name === 'ssr') return `ssr.cjs`;
 
-							return `${entry.name}/index.${ext}`;
-						}
-					},
-					chunkFileNames: `internal/[name]-[hash].${ext}`,
-					format,
-					minifyInternalExports: false,
-					dir: '.',
-				};
-			}
-		),
+					return `${entry.name}/index.cjs`;
+				}
+			},
+			chunkFileNames: `internal/[name]-[hash].cjs`,
+			format: 'cjs',
+			minifyInternalExports: false,
+			dir: '.'
```

Comment:
do you think we could put it in `dist` instead? It's rather messy to me when the root directory locally is filled with built files

Reply:
No, because we need the type definitions mirroring that structure, as we can't assume everyone uses `moduleResolution: bundler`, so we might as well put it in directories along with the cjs version.

Reply:
I don't quite understand. Why can the ESM version be located in a sub-directory but the CJS version can't?

Reply:
Actually, since we're getting rid of the CJS version, I guess this is point is moot